fstar-helpers/Makefile.generic

```diff
@@ -120,23 +120,19 @@ endif
 
 .PHONY: all verify clean
 
+# Only run code extraction for build targets, not for 'clean'.
 all:
+ifeq "$(wildcard *.fst *fsti)" ""
+	$(shell cargo hax into fstar)
+endif
 	$(Q)rm -f .depend
 	$(Q)$(MAKE) .depend hax.fst.config.json verify
 
 all-keep-going:
-	$(Q)rm -f .depend
-	$(Q)$(MAKE) --keep-going .depend hax.fst.config.json verify
-
-# Only run code extraction for build targets, not for 'clean'.
-all all-keep-going:
 ifeq "$(wildcard *.fst *fsti)" ""
 	$(shell cargo hax into fstar)
 endif
 	$(Q)rm -f .depend
-all:
-	$(Q)$(MAKE) .depend hax.fst.config.json verify
-all-keep-going:
```

Comment:
Is that a conflict with https://github.com/cryspen/libcrux/pull/1101 or something? or it is intentional?

Reply:
not intentional, was a conflict. fix if incorrect?